frontend/messages/cs.json

```
@@ -1179,6 +1179,294 @@
+	"close": "Zavřít",
+	"collapseAll": "Sbalit vše",
+	"confirm": "Potvrdit",
+	"copied": "Zkopírováno!",
+	"delete": "Smazat",
+	"expandAll": "Rozbalit vše",
+	"exportYaml": "Exportovat",
+	"implementationGroups": "Implementační skupiny",
+	"no": "Ne",
+	"number": "Číslo",
+	"publish": "Publikovat",
+	"requirement": "Požadavek",
```

Biome (2.4.15)

[error] The key requirement was already declared.

(lint/suspicious/noDuplicateObjectKeys)